### PR TITLE
Add group to get SDL validation params.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
DevDiv changed the bug area path.
@sunandabalu created a variable group to use.

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
@sunandabalu I'm not all that good with yaml, could you check this for me and see if I added the group  correctly and that I used the proper variables. Thanks.

File: azure-pipelines-arcade.yml [hunk 1]
```diff
@@ -273,6 +273,8 @@ stages:
 - ${{ if eq(variables._RunAsInternal, True) }}:
   - template: eng\common\templates\post-build\post-build.yml
     parameters:
+      variables:
+        - group: SDL_Settings
```
Comment (sunandabalu) [inline azure-pipelines-arcade.yml]: This variable group has values specific to Arcade, please do not use this. Create one of your own.

Comment (sunandabalu) [reply azure-pipelines-arcade.yml]: And variable group needs to be added here - https://github.com/dotnet/wcf/blob/9c7d5d85d7466871f4c8d2d85473301443e48564/azure-pipelines-arcade.yml#L34

Comment (StephenBonikowsky) [reply azure-pipelines-arcade.yml]: Thanks for your assistance @sunandabalu 
Added a Wcf group for the SDL validation params and amended the commit.

**Comment by sunandabalu** [comment]
LGTM

**Comment by StephenBonikowsky** [comment]
CI failure is understood, not related to these changes.

**Pull request merged**